react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,24 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
```

Comment:
Do you need to use the spread here?

Reply:
@LucasZibell yes, because it's necessary pass this test:
> min can return the minimum based on many parameters